packages/libs/error-stack/src/ext/stream.rs

```diff
@@ -0,0 +1,368 @@
+//! Implements support for `error-stack` functionality for streams. The main trait in this module
+//! is [`StreamReportExt`] (which is so named to avoid conflict with other stream-related types
+//! common to the Rust `async` ecosystem).
+//!
+//! It may be helpful to first read the [`ResultExt`] documentation.
+use core::{
+    fmt::{Debug, Display},
+    task::Poll,
+};
+
+use futures_core::Stream;
+
+use crate::{Context, Report, ResultExt};
+
+/// Extension trait for [`Stream`] to provide contextual information on [`Report`]s.
+///
+/// [`Report`]: crate::Report
+pub trait StreamReportExt: Stream + Sized {
+    /// Adds a new attachment to the [`Report`] inside the [`Result`] when
+    /// calling [`Stream::poll_next`].
+    ///
+    /// Applies [`Report::attach`] to the [`Err`] variant, refer to it for more information.
+    ///
+    /// [`Report`]: crate::Report
+    /// [`Report::attach`]: crate::Report::attach
```

Comment:
It appears that these links are not needed. Could you also check this for other extensions please?